### DISCUSSION TITLE
How to obtain the rough shape information of the object by voxel occupancy of the grid

### DISCUSSION THREAD

**Discussion by kevincheng3** [google-deepmind/mujoco#2031] [Asking for Help]
Hi,

I'm a student  and I'm trying to use MuJoCo for robotic hand manipulation.

I'm looking for some help with obtain the rough shape information of the object by voxel occupancy of the grid. We believe that coarse object information can help train a controller capable of manipulating different objects

As shown in the image below, we attach the grid to the hand. In practice, we will have a larger number of finer grid. 

**### Our problem is how to obtain the grid occupied by the object during the actual manipulation in real time？**

Here is a screenshot / video, illustrating my question:
![issure1_1](https://user-images.githubusercontent.com/43990826/235039816-b270837a-91d2-4a10-a82f-76efdc4df6de.png)

![issure1_2](https://user-images.githubusercontent.com/43990826/235039857-2908e23f-784f-4a91-993a-664197cf0d6a.png)

**Comment by yuvaltassa** [comment]
Could you please explain further?

As far as I can tell, you want to find out the occupancy of points of interest in some voxel grid that you define yourself. Why is this not a trivial thing? Did you try to do this and fail? What is the blocker?
